cartography/intel/aws/resourcegroupstaggingapi.py

```diff
@@ -153,14 +155,28 @@ def load_tags(
         resource_label=TAG_RESOURCE_TYPE_MAPPINGS[resource_type]['label'],
         property=TAG_RESOURCE_TYPE_MAPPINGS[resource_type]['property'],
     )
-    neo4j_session.run(
+    tx.run(
         query,
         TagData=tag_data,
         UpdateTag=aws_update_tag,
         Region=region,
     )
 
 
+@timeit
+def load_tags(
+    neo4j_session: neo4j.Session, tag_data: Dict, resource_type: str, region: str,
+    aws_update_tag: int,
+) -> None:
+    neo4j_session.write_transaction(
```

Comment:
curiosity: why is it write_transaction but not read_transaction(if there's such thing?), this method is load tags and sounds to be read only

Reply:
There is a read_transaction. The difference is when there is a clustered environment, with different hosts designated for reads and different hosts designated for writes.

But load_tags does indeed write to the database.